app/src/test/java/com/github/se/signify/model/navigation/NavigationActionsTest.kt

```diff
@@ -0,0 +1,96 @@
+package com.github.se.signify.model.navigation
+
+import androidx.navigation.NavHostController
+import androidx.navigation.NavOptionsBuilder
+import com.github.se.signify.model.auth.MockUserSession
+import org.junit.Before
+import org.junit.Test
+import org.mockito.kotlin.any
+import org.mockito.kotlin.anyOrNull
+import org.mockito.kotlin.eq
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.never
+import org.mockito.kotlin.verify
+
+class NavigationActionsTest {
+
+  private lateinit var navController: NavHostController
+  private lateinit var userSession: MockUserSession
+  private lateinit var navigationActions: NavigationActions
+
+  @Before
+  fun setUp() {
+    navController = mock()
+    userSession = MockUserSession()
+    navigationActions = NavigationActions(navController, userSession)
+  }
+
+  @Test
+  fun test() {
```

Comment:
I dont think this naming is intended right? Its not aligned with the rest.

Reply:
Good catch. I forgot to delete this temporary test of a test.